app/src/main/java/org/simple/clinic/recentpatient/RecentPatientScreenItemTypes.kt

```diff
@@ -26,7 +29,8 @@ data class RecentPatientItem(
     val name: String,
     val age: Int,
     val lastBp: LastBp?,
-    val gender: Gender
+    val gender: Gender,
+    val latestUpdatedAt: Instant
```

Comment:
The fact that this `Instant` is the latest updated at timestamp selected from multiple timestamps is an implementation detail that is known to the `RecentPatient` model, but need not be known here. Rename this property to just `updatedAt`?

Reply:
done